### ISSUE TITLE
No version variables on windows

### ISSUE THREAD

**Issue by TurkeyMan** [SaturnSDK/Saturn-SDK-GCC-SH2#6]
Trying to run `build.sh` on windows, but none of the tool version variables are set.
Can you include a script that sets the variables to appropriate (known working?) versions?
`${BINUTILSVER}`, `${GCCVER}`, etc... I have no idea what version numbers to punch in there..

Also, is it safe to say I should use `x86_64-w64-mingw32` for `BUILDMACH` and `HOSTMACH`?

Perhaps it would be good to include a build-win.sh script which sets all the required variables for windows prior to running `build.sh`?

**Comment by TurkeyMan** [comment]
Just tried to build from linux, and I'm missing these variables too... some guide what versions to use would be helpful.

**Comment by RedRingRico** [comment]
I guess it would have helped if I had mentioned the name of the build script in the `README`. It's `build-elf.sh`, which has all the correct variables set for version numbers.

For a 64-bit Window build, that tuple should be correct. I have yet to test compilation on any platform other than GNU/Linux, though MSYS2 would be your best bet on Windows.

**Comment by TurkeyMan** [comment]
Ah wow, I can't believe I missed that! That said, the filename is not intuitive ;)
I'll give it a go using msys2 and let you know how it goes...

**Comment by RedRingRico** [comment]
Yeah, it's a relic from when there was `build-coff.sh` and `build-elf.sh`.

**Comment by TurkeyMan** [comment]
```
...
checking size of void *... 4
checking size of unsigned short... 2
checking size of unsigned... 4
checking size of unsigned long... 4
checking size of mp_limb_t... 0
configure: error: Oops, mp_limb_t doesn't seem to work
make: *** [configure-gmp] Error 1
Failed building the bootstrap phase of GCC
Failed to build the ELF toolchain
```
It built quite a lot of stuff, but died kinda early.

**Comment by TurkeyMan** [comment]
MINGW64, MSYS2, target tuple as I said at the top :(

**Comment by RedRingRico** [comment]
It seems to be an issue with using absolute paths. I’ll look into it.

**Comment by TurkeyMan** [comment]
Cheers

**Comment by TurkeyMan** [comment]
Did fix?

**Comment by RedRingRico** [comment]
This should be fixed as of commit bff0b60e26d958464b089a932f46324d4fbd9760.
